.github/scripts/trymerge.py

```diff
@@ -1116,15 +1116,20 @@ def gen_commit_message(
         msg = self.get_title() + f" (#{self.pr_num})\n\n"
         msg += msg_body
 
-        # Mention PR co-authors
-        for author_login, author_name in self.get_authors().items():
-            if author_login != self.get_pr_creator_login():
-                msg += f"\nCo-authored-by: {author_name}"
-
         msg += f"\nPull Request resolved: {self.get_pr_url()}\n"
         msg += f"Approved by: {approved_by_urls}\n"
         if ghstack_deps:
             msg += f"ghstack dependencies: {', '.join([f'#{pr.pr_num}' for pr in ghstack_deps])}\n"
+
+        # Mention PR co-authors, which should be at the end of the message
+        # And separated from the body by two newlines
+        first_coauthor = True
+        for author_login, author_name in self.get_authors().items():
+            if author_login != self.get_pr_creator_login():
+                if first_coauthor:
+                    msg, first_coauthor = (msg + "\n", False)
```

Comment:
Is this adding two lines before the list?
Can this be simplified by doing something like `msg += "\n\n"` before the loop and removing the `first_coauthor` logic?

Reply:
There might not be any co-authors and in this case it will add two empty lines at the end of the message. But I can add `.strip()` to mitigate that